ansible/roles/base/tasks/main.yml

```diff
@@ -17,19 +15,24 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
+      apt: 
+       name: python3-pip
       become: yes
+      become_method: sudo
 
-    - name: install python-virtualenv
-      apt: name=python-virtualenv
+    - name: Install python-virtualenv
+      apt:
+       name: python-virtualenv
       become: yes
 
     - name: Install PIP requirements
+      become: yes
+      become_method: sudo -H
       pip:
         requirements: /vagrant/ansible/pip-requirements.txt
-        virtualenv: /home/vagrant/python3.6
+        virtualenv: /vagrant/python3.6
```

Comment:
Hi Rob, Thanks for the honest and detailed comments.
Reason i changed the virtualenv to /vagrant/ was to sync the files back to my local so i can commit.
Also Zappa needs to run inside the virtualenv, hence the project was put inside the python3.6 folder or should we create the virtualenv outside the python3.6 folder?